### DISCUSSION TITLE
http_client request count not counting client requests properly and not showing calls in toolbar

### DISCUSSION THREAD

**Discussion by oplaza** [symfony/symfony#49463] [Q&A]
When making requests with httpclient for clients not in the default client, the request count is not counting properly and that provoques not to showing the calls in the service.
this is what i have:

![image](https://user-images.githubusercontent.com/12097122/220304578-3b895486-3d33-4b1f-a0d5-84b567e8ed11.png)

if i remove the code that has the request count in the profiler http_client twig:
![image](https://user-images.githubusercontent.com/12097122/220304781-cb2b98f2-d9ae-4c69-afa6-8bc2c22e7c0a.png)

it works as it should:
![image](https://user-images.githubusercontent.com/12097122/220305039-76295ff2-0cd3-4613-bc6f-ccc1c614b539.png)
as u can check it says total requests 0 but i have 2 in one of the clients.

**Comment by MatTheCat** [comment]
Which version of Symfony are you using? This looks like a duplicate of https://github.com/symfony/symfony/issues/49219.
